packages/babel-helpers/src/helpers/dynamicImportSpecifier.js

```diff
@@ -0,0 +1,9 @@
+/* @minVersion 7.20.6 */
+
+export default function _dynamicImportSpecifier(s) {
+  try {
+    return Promise.resolve("" + s);
+  } catch (e) {
+    return Promise.reject(e);
+  }
```

Comment:
```js
  return new Promise(function (resolve, reject) {
    try { resolve("" + s) } catch (e) { reject(e) }
  })
```
will be 4 bytes shorter after terser since the it can mangle names like `resolve` and `reject`.

Reply:
```js
return new Promise(resolve => resolve(""+s));
```

also works. It's simple enough that we probably don't need an helper for that.

Reply:
Oh, wait, maybe we don't even need a helper. Since the `f` in `new Promise(f)` is executed synchronously, we just need to do this:

```js
import(s)
```
=>
```js
new Promise((resolve) => resolve(require("" + s)))
```

And throwing in `toString` can automatically make a rejected promise.

Reply:
I still want to keep `require` in `.then`, so that the imported code is executed on the next tick.